### ISSUE TITLE
99% of the community you want to reach will never read anything on github

### ISSUE THREAD

**Issue by zookozcash** [ZcashFoundation/ZcashFoundation#58]
Dear Foundation:

A couple of times in the past, I've mentioned to some of you in conversation that I had talked to someone (a non-computer-programmer) who said something like "I was interested in this Zcash community and governance thing, but then I tried to join in but it was on github, so I gave up".

Each time I've told you this, whichever one of you it was that I was talking to said something that showed that you just didn't get it. Like "What's wrong with github? Why did they give up?" or "Maybe we could add navigation or instructions or something".

So, I was frustrated, but it's your call so I didn't want to bug you about it.

But now it has happened again. For what I think is the 4th time, I was talking to someone who was a non-computer-programmer (this person is a journalist), and as I always do I emphasized the importance the inclusivity of the Zcash Foundation, and she replied "Oh yeah, I tried to look into that thing, but it was on github and I can never find anything on github."

So, please allow me to be clear and forceful about this:

_As long as participating in the Zcash Foundation, expressing your voice, or exercising your power, requires using github, then it will be limited to computer programmers only._

There is no way to solve this with education or with work-arounds. The only way to include non-computer-programmers in your intended inclusive community is to make it possible to have the complete, fully-empowered experience of participating in that community, making your voice heard, observing and responding to the results, exercising power, without ever logging into github or searching for documents within github.

**Comment by prahaladbelavadi** [comment]
Given that tons of resources are available for learning how to use and understand github workflow, its the leap in adoption curve for non developers. 
We could put together a bunch handy links, but eventually it's a personal decision.

Here's a bunch of links:

* [Github Guides](https://guides.github.com/activities/hello-world/)

* [Github Youtube](https://www.youtube.com/githubguides)

* [A list of Git tutorials](https://gist.github.com/jaseemabid/1321592)

* Simplest intro to git by [github](https://github.com/) and
  [codeschool](http://codeschool.com/) - [Try Git](http://try.github.com/)

* [PeepCode Git Internals PDF](https://github.com/pluralsight/git-internals-pdf)

* [Intro to github]
  (http://gun.io/blog/how-to-github-fork-branch-and-pull-request/)

* A very simple tutorial to start with.
  [Intro to git for web designers](http://www.webdesignerdepot.com/2009/03/intro-to-git-for-web-designers/)

* [Git Visual Guide](http://marklodato.github.io/visual-git-guide/index-en.html)

* [Think Like (a) Git](http://think-like-a-git.net/). A series of introductory
  blogs, aimed at beginners.

* [Everyday GIT With 20 Commands Or So](http://www.kernel.org/pub/software/scm/git/docs/everyday.html)
  learn Git basics through the most common commands with examples

* [Git for the lazy](http://www.spheredev.org/wiki/Git_for_the_lazy). A wiki
  with a bunch of good notes.

* [Help.Github](http://help.github.com/)

* *The man pages*. Its the definitive guide to learn the internals. Its
  available in the console as `$ man git` or `$ git --help`. Detailed
  information about a particular command can be accessed with `$ git commit
  --help`.

* A very decent book on git. Spend some time and read it cover to cover.
  [Pro Git book](http://git-scm.com/book)

* [Git from bottom up](http://ftp.newartisans.com/pub/git.from.bottom.up.pdf) is
  another good book

* [Git Ready](http://gitready.com/) A collection of blogs, each concentrating on
  one git command at a time. Difficulty level varies from very beginner to
  advanced.

* [Git magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/) is an online
  book on Git well written with wonderful analogies to help beginners understand
  the concepts behind Git

* [A Note About Git Commit Messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
  A must read on writing good commit messages and the style.

* [Seth Robertson's blog](http://sethrobertson.github.com/) gives some real
  insight into git internals and good practices

* [Git Immersion](http://gitimmersion.com/) walks you through Git basics using a
  step-by-step, hands-on approach. It’s a great tutorial for those who prefer to
  get into Git right away.

* [Github Cheat-sheets](http://help.github.com/git-cheat-sheets/) Cheat sheet
  for quick reference

* [Git-fu](http://gitfu.wordpress.com/). Intermediate blogs

* [Git is simpler](http://nfarina.com/post/9868516270/git-is-simpler)

* [Mastering Markdown](https://guides.github.com/features/mastering-markdown/)

*[Learn git through Udacity](https://blog.udacity.com/2015/06/a-beginners-git-github-tutorial.html)

Cheers!

**Comment by mineZcash** [comment]
What style of interaction would be easier for the casual user? A discourse forum like the Zcash forum perhaps?

**Comment by john-light** [comment]
I am a non-programmer who has no problem using GitHub. It's even easier than Facebook if you design the workflow with non-programmers in mind. IMO the laundry list of resources above is not helpful as it could easily lead to "analysis paralysis". I think a simple and quick guide specifically for "Participating in the Zcash Foundation on GitHub" would be much more helpful for the uninitiated. This guide by Enspiral is a good _example_ that could be followed: https://handbook.enspiral.com/guides/github_for_beginners.html

**Comment by tloriato** [comment]
I **love** GitHub for every reason that I assume you all do. _Probably more._

It has become clear for me, however, that GitHub is a barrier for non-programmers. It **really** is. Last week we had a conversation about this in my workspace about how programmers and "stakeholders" could interact more and a platform to bring them together. 

Without going into the details of the discussion, the fact remains: **Trying to bring non-programmers into GitHub is like trying to create a cypherpunk community on Facebook.**

I'd love to see a more throughout discussion about platforms more easily reachable by everyday people. It could be more Forum-alike (http://flarum.org/) or the classic Slack/Rocket.Chat.

edit: That being said, if we _were_ to stick with GitHub I'd feel that the above suggestion from john-light is the way to move forward: Making specific guides/tutorials to interact with us in here. But don't go making a 20 min video, no one is going to watch and we'll come back to the same problem.

**Comment by zookozcash** [comment]
I love github, too.

But if you want to achieve inclusivity, you need to know educating the people before they can engage is just a non-starter. The people that you want to include will refuse even to read the educational material and learn the new skills.

If you're okay with excluding those people, that's a different conversation. But if you want to include those people, then you be aware that this is not an option that accomplishes that.

**Comment by amiller** [comment]
Thanks for posting this issue!

Here are a few potential options I have in mind: 
1. We've recently created a mailing list, https://lists.z.cash.foundation/mailman/listinfo/general . This allows people to post without signing up for github (although they have to sign up to a mailing list, which I think might be as difficult.)
2. We can use the existing zcash forum. There's a Foundation folder in there. https://forum.z.cash/c/foundation  We haven't been too diligent about mirror content on that forum though.
3. We've experimented with a "deliberation" service called kialo. We got a trial invitation token for anyone in the Zcash community to try it out. https://www.kialo.com/invited?token=005-4771917b-de9d-4258-bdd9-e3356589b0d2 One problem with it is it doesn't seem so easy to archive publicly, requiring a log-in. In any case I don't think we've gotten much feedback on it yet.



What else should we try?

**Comment by diracdeltas** [comment]
in terms of lowest barrier to entry, i think Slack/Rocketchat wins. (i slighly prefer Slack since i usually have it open anyway)

the downside is that Slack history search/organization is pretty bad. so any important decisions/proposals should probably be mirrored to github/gdocs.

**Comment by tloriato** [comment]
+1 for Kialo, seems really awesome. Would love to try it out!

**Comment by imichaelmiers** [comment]
I strongly disagree with this point.  First, as others point out github is simple enough to use. Second, there is considerable cost to setting up alternatives  and that cost shouldn't be born now when its critical to focus on expanding zcash usage and other issues.  And thats just looking at the technical cost, consider the issues with trolling that arise to varying degrees across platforms. To pick a  obvious straw man, consider the time eaten up if governance  was done via twitter.  Finally, the barrier to entry in participating  in governance discussions need not and  should not be lower than using zcash itself  (in fact i'd argue it should be slightly higher).  Github is well above that threshold currently.

**Comment by jrmoreau** [comment]
Chats are chaos. Totally overwhelming and not efficient for archiving/categorizing relevant content and convos. I'd suggest having a helpdesk/support email with a knowledge base. Having a single full-time person to answer high level questions seems well within the foundation's capacity and would provide a HUGE value add to the community.

**Comment by mineZcash** [comment]
There currently is a Rocket.chat channel dedicated to interacting with the Zcash foundation which all the board members (Andrew, Josh, Tromer, etc) frequent:

https://chat.zcashcommunity.com/channel/the-zcash-foundation

A new person who isn't comfortable with GitHub can get also get answers there if they need them.

**Comment by acityinohio** [comment]
Above and beyond @amiller's point, and the great discussion here, I don't think this is purely a Zcash Foundation issue. This is an open tech governance issue (as someone tweeted in response to you, @zookozcash, after you announced posting of this issue):

https://twitter.com/nuttycom/status/966382282406739968

I'm sorry you don't feel like you've been heard @zookozcash, and for what it's worth I place the blame squarely on me. Truth be told, I (and I'm sure others in the Foundation) have been listening to the feedback, but it hasn't been a priority to shift to other methods because the at its most abstract the problem—at least to me—feels like a broader open source issue, one that is not easily solved (per the tweet above). I apologize if I didn't state this explicitly previously.

Meanwhile, for our existing mission—and the roadmap [we've outlined here](https://github.com/ZcashFoundation/ZcashFoundation/blob/master/2018-VISION.md)—GitHub and the listservs serve as reasonable forums for anything technically oriented. And as @amiller suggested, we've been experimenting with other methods for debate and collection, like Kialo.

But to get at the core of it, let's return to your explicit call-out:

_As long as participating in the Zcash Foundation, expressing your voice, or exercising your power, requires using github, then it will be limited to computer programmers only._

The [governance proposal outlined by Andrew](https://github.com/amiller/ZcashFoundation/blob/election/election-proposal.md)—while submitted via GitHub, like this issue—advocates that the new board be selected by a broad community panel, where **votes can be collected by any medium preferred by the panel** (see this [section here](https://github.com/amiller/ZcashFoundation/blob/election/election-proposal.md#community-approval-panel)). 

As it stands today—given the way the Foundation is structured as a non-profit—the Board is still the ultimate authority, and there is no better way to **exercise power** than appointing members. In my view, @amiller's most recent proposal is likely the _most_ inclusive one can possibly get in practical terms.

And when it comes to expressing one's voice...well, you posted this on Twitter. You could have posted a text file on Twitter, and it would have engaged discussion. There's a mailing list. There's the community rocket chat (which as @mineZcash notes, we frequent), there are forums (which I frequent), and we will try to stay abreast of them all. We can attempt to wrangle and manage them all, but generally GitHub is the easiest way to make _our_ operations transparent, but maybe the General mailing list is a better place for that in the future...and in any case, we can attempt to provide feedback in all the places we can. Discussion doesn't happen in one place, and whatever our "official" place for providing transparent feedback on the Foundation's operations, we can always provide feedback elsewhere (but maybe we could do a better job there too).

In tactical terms, the website will be improved this year and there's an opportunity to collect feedback there as well.  Moreover, if a journalist is who you're talking to, and they are still confused about the Foundation, you can always send them my way: josh [at] z [dot] cash [dot] foundation. This is obviously not scalable/a bandaid, but I suggest it all the same.

Anyway, to summarize:

- This is an open open source question, and not unique to the Zcash Foundation
- Even so, we've been opening up other avenues for feedback/experimenting with other options...
- In its most fundamental terms, @amiller's governance proposal (which defines how community members can exercise power) **DOES NOT require GitHub**
- GitHub still works very well for providing transparent access to our work (but maybe not for providing feedback from the non-technical)
- We will be doing more tactically this year to widen the net for feedback
- We hear you @zookozcash

**Comment by diracdeltas** [comment]
@zookozcash: I'd like to request that the next time someone complains that Github is a barrier to their participation, please ask them (1) what the specific problem is, and (2) what they would rather use. Without that data, we're going to have a hard time coming up with a satisfactory solution.

**Comment by apertamono** [comment]
@prahaladbelavadi Thank you very much. Those are very useful resources for me, because I'm a former programmer, honorably discharged, who doesn't begin to understand how to rebase or squash commits.

Sorry for barging in, this isn't my community, I'm used to the lively chaos of you-know-who, the evil drug-dealing cryptojackers, who have their own communication problems, but by any objective measure do a better job of reaching users and recruiting contributors. I wouldn't call this a community at all if you have to drag people in to make them read the Foundation's documents. For example, I took a look at the translation plans, and apparently some apparatchik has decided which languages to translate to first...

Anyway, @zookozcash is a hypocrite, because he aborted the Zcash subreddit with his own hands.

**Comment by burdges** [comment]
It's true github's interface sucks less than afaik all existing forum software.  As an intermediate step, you might host a blog that maintains short summaries of discussions on github with links to the specific discussions, except there remains one major problem with asking less tech savvy users to make github accounts:  

_A default github configuration acts like a mailing list subscription and floods your mailbox._

I'll therefore make two suggestions: 

First, has anyone considered using open source github clones like gitlab, etc.?  It's true a gitlab, et al. interface is similarly designed for programmers, and they lag behind github in spam management, but you can easily configure one not to be the spam factory github itself is, and over time you could simplify the interface. 

Second, I've participated in W3C that employed some strange merger of github, mailing lists, and maybe another comment entering scheme.  In fact, these W3C group's approach sucked because participants get spamed the messages in multiple ways, but one could provide another non-github interface to github discussions via SMTP that provides a simpler interface and does not spam users.

**Comment by daira** [comment]
The problem with most forum software is that it's designed for ephemeral discussion. An issue tracker, on the other hand, is designed for permanently archived discussions, which is absolutely needed for governance.

The technical requirement here is for an issue tracker. Anything that tries to be non-programmer-oriented but is still up to the task is going to have to implement the features of an issue tracker. It's hard to interest developers of general forum software in implementing those features, which are not needed for the majority of forum use cases.

So, while I agree that there's a serious and real obstacle to non-programmers perceiving GitHub as an appropriate medium, it's going to be very hard to come up with something that is not strictly worse, IMHO.

**Comment by Tectract** [comment]
> As long as participating in the Zcash Foundation, expressing your voice, or exercising your power, requires using github, then it will be limited to computer programmers only.

At the Heartbeat Coalition (heartbeatcoalition.org), we are using the membership dues to basically pay for the github "seats" for each user, which then allow those users to access the shared kanban boards that we talk about and use together for project planning and during our teleconference screenshares. It's a way of bringing non-programmers into the programmer-space :)

**Comment by LKNX** [comment]
Why dont you start using https://steemit.com for communication? Almost 1 Million crypto-users there, all of them have money - great place to keep a community and it is free.

**Comment by ChaceHunter** [comment]
**99% of the community you want to reach need to know much more than GitHub.**

I am your non-computer-programmer, *some college*-educated average individual that believes in, and uses Zcash. In order to use Zcash to its full potential, I had to:

- download and install Linux
- spend days learning the CLI
- spend days with `zcashd` to get the full node up and running
- spend days learning the `zcashd` CLI
- find out that I couldn't send ZEC from my Ledger's t-addr to my z-addr
- create a t-addr using `zcash-cli`, send ZEC from Ledger to that t-addr, send ZEC from that t-addr to my cozy z-addr
- spend days learning how to send an encrypted memo. This involved:
     - poring over the existing documentation with 20+ tabs open at a time
     - seeking help on the Zcash Community Rocket Chat only to find out that all the information presented to me resulted in JSON parsing errors
     - Only when I followed the instructions *exactly* as they were written in `zcash-cli z_sendmany help`, (something that still wasn't inherently intuitive to me at the time), was I able to get my transaction to go through

Oh, and guess what? During that time I didn't know how to copy/paste in XTerm so I was *hand-typing* all the addresses! WTF, how am I this much of an idiot?!

Now, I would do it all over again. I'm not complaining. I have since been the first (and only?) person to buy something on OpenBazaar using ZEC. I reached out to other merchants asking if they were going to accept ZEC now that OB integrated, but the ones that responded expressed their disappointment that they would have to run an entirely separate node from what they were already running. I'm getting beside the point..

I say all this because despite all that time learning the intricacies of Zcash, **it was never clear to me that I could participate in Zcash Foundation discussions until now.** I didn't even know that discussions were being held here. I *knew*, but I didn't **know**, you know? That could just be a failure on my part, or something I could have easily overlooked in my quest to get some some goddang ZEC stickers to put on my laptop and bike. For all I know, there could be a drastically simpler way to use shielded addresses. But from the beginning of this quest to solidify my foundational understanding of Zcash, nothing has really been.. *clear.*

There are new people entering this space by the thousands everyday. For them, it must seem like a cacophonous maelstrom of information that sweeps them up before they can ever get their feet on the ground. To find the necessary confidence to *actually* voice one's opinion requires time and education. It took me 2 years before I finally felt 'leveled up' enough to even think about participating in higher-level discussions without the shroud of self-doubt trying to prevent me from doing so.

As @acityinohio mentions, this is definitely a broader open source community issue, but by continuing to practice inclusiveness, by being aware of these problems, and by having these public discussions, I think we're on the right track.

I hope I've given you a better understanding of what it takes 99% of the people you want to reach to *get there*.

Thank you all,
Chase

**Comment by burdges** [comment]
Aside from gitlab, et al. and github front-ends mentioned in https://github.com/ZcashFoundation/ZcashFoundation/issues/58#issuecomment-367518475, you might evaluate the open source stack exchange clones and the liquid democracy software, which presumably both manage long term discussions too.  I think open source stack exchange clones will prove too gamified in the wrong direction, but still worth considering.  I think the liquid democracy software was designed precisely to be an issue tracker for governance, but it may posses unexamined failings, well the Germany Pirate Party largely failed so far.

**Comment by burdges** [comment]
Also, you'll find more information on liquid democracy at https://liqd.net/en/software/ or even https://en.wikipedia.org/wiki/Delegative_democracy or https://medium.com/organizer-sandbox/liquid-democracy-true-democracy-for-the-21st-century-7c66f5e53b6f

**Comment by zookozcash** [comment]
I mentioned this topic to a Chinese person yesterday and they reminded me that github is blocked in China.

**Comment by john-light** [comment]
@zookozcash to be fair, that is a problem with GitHub specifically, not necessarily with the git/ issue-tracker model of collaboration, and so orthogonal to your original concerns (I think) i.e. Zcash Foundation could switch to a self-hosted GitLab instance and still have the perceived higher barrier to participation even though people in China could access it as long as it is unblocked.

**Comment by xaur** [comment]
China can block anything including GitHub. Someone told me it works time to time. And depending on location, the blocking is more or less severe.

**Comment by xaur** [comment]
> "I was interested in this Zcash community and governance thing, but then I tried to join in but it was on github, so I gave up".
> "Oh yeah, I tried to look into that thing, but it was on github and I can never find anything on github."

This is confusing.

* Zcash has RocketChat, IRC, forum, Reddit. Why did they rush to GitHub? Perhaps some page wrongly points them here?
* What exactly were they trying to do? They don't need Git, GitHub workflow, or even Markdown knowledge to login and post comments.

It is important to consider _who_ to include into _what_. There are several levels of participation. Higher levels naturally have higher barriers.

To onboard new people and help them out, chats work just fine: Slack, Discord, Matrix, RocketChat, IRC and such. Casual platforms like Reddit and Facebook also fit. "I tried to look into that thing" falls into this category.

Next, when they get around and have something to say, they have a choice. If it's something quick and not too important they can use chat. If they want to start a thoughtful discussion that won't drown in history, they better use something topic-based: GitHub, Reddit, forum, mailing list. Commenting there is no more difficult than posting on Twitter or Facebook. Login, search, click, enter text, submit are trivial actions. I would be surprised if a person having something to say about Zcash cannot accomplish them. Further, if they can _use_ Zcash, they can surely handle GitHub.

Next level if they have something to contribute. Good idea, bug report, code, design, and such. GitHub is necessary at this level.

Highest level is "exercising power". This is governance. This is serious. This level demands more from the person _and_ communication software in use.

The person must know what they are doing, have some skills and skin in the game -- reputation or wealth to loose. They must be willing to do whatever is necessary. I think you don't want people incapable of logging into GitHub excercising power over Zcash. Ideally they hold ZEC (skin in the game).

If we are talking about open governance in open source project, communication software used for hosting governance discussions and decisions must satisfy special requirements, and GitHub does not pass them.

@daira all issue trackers I know do not enforce permanent archiving in any way. GitHub can delete comments any moment. (It usually doesn't, and I love it, but it _can_). There's no easy way to fetch GitHub issue threads locally. Their integrity and authenticity is not protected like in Git repos. Distributed issue trackers exist ([Fossil](https://en.wikipedia.org/wiki/Fossil_%28software%29)) and worth looking at. But governance needs even more.

This is what communication software used for governance must do:

* All messages are timestamped and signed to verify integrity, the whole repository is versioned like Git.
* Decentralized like Git, or at least trivially replicatable. Must survive GitHub shutdown or censorship with no data loss and minimum time to recover.
* All "good" messages are stored forever.
* Balanced approach to "bad" messages: Transparent, publicly provable censorship to strip outright spam (that _will_ come). At the same time, prevent silent censorship like on Twitter and Reddit.
* Tiny payment barriers to register and post to further discourage garbage. Note, we're talking about _tiny_ amounts to combad spam, not exclude people.
* It is _very_ expensive to overwrite content, which is achieved by anchoring to some public blockchain.
* For projects funded by cryptocurrencies: ideally, disbursements are automated and control is decentralized.
* Special case for cryptocurrency projects: holders must have the power.

As @acityinohio notes this is broader issue of governance in open source projects that is not easily solved. Luckily, I see a solution coming.

A system in development called Politeia ([intro](https://blog.decred.org/2017/10/25/Politeia/), [code](https://github.com/decred/politeia)) fits all above requirements. It is built by Decred project that explores governance all the way down the rabbit hole, but it is generic by design -- everything Decred-specific is a plugin. (In fact it is generic enough to support many [usecases](https://btcmanager.com/decreds-politeia-challenge-overview-winners/)). I think it can be easily customized for Zcash: use ZEC for paywalls, anchor to Zcash blockchain, collect holder opinion, pay from Zcash fund, and so on. Disbursement part is not there yet but it is a must and is planned. It is somewhat similar to Vitalik's DAICO but the whole thing is broader in scope of processes it covers.

Regarding the last requirement, "holders must have the power", I'm not sure if Zcash is willing to go that route. But even without it, I believe all other properties are very desirable for many open projects.

**Comment by prahaladbelavadi** [comment]
Hey @mineZcash : https://github.com/ZcashFoundation/ZcashFoundation/issues/58#issuecomment-367432092

I think a discourse or a disqus platform would probably help answer FAQs while allowing the community to collaborate and answer each other's questions, preventing confusion to a large extent.
Having an established well trusted web presence is quite essential and gravity of the situation grows with increasing number of scams in this sector.
[Ben Tossell](https://twitter.com/bentossell) has been instrumental in building the ecosystem with token daily and his posts on twitter. He'd have some valuable insights on this.

Hey @ProkhorZ : https://github.com/ZcashFoundation/ZcashFoundation/issues/58#issuecomment-367506069

Glad to be of help :)

**Comment by zookozcash** [comment]
Here's another example of someone who wants to influence the course of the Zcash community (they are concerned about the development of ASIC miners and are advocating for changing the PoW), but who refuses to go to github: https://forum.z.cash/t/let-s-talk-about-asic-mining/27353/33?u=zooko

"Since this is the offical ZCash forum, I’m expecting that posts here are “on the Foundation’s radar.” Github is too complicated for me. They should - and must - watch this discussion carefully."

**Comment by sonyamann** [comment]
I'm a little late to the discussion, but I want to chime in. (For those who don't know, I've newly joined the Zcash Foundation to run communications.)

I agree that GitHub is not a suitable default hub for anyone nontechnical. Usability, especially as a beginner, is a huge hurdle in the cryptocurrency world overall. We definitely want to work on lowering the barriers; that's something I care about a lot.

Personally, I don't think GitHub needs to be replaced altogether in the Foundation's workflows, since it's a valuable tool and one that many F/OSS participants are familiar with. But it shouldn't be the only participation venue maintained by the Foundation.

One of my near-term goals is to build out the resources on the Zcash Foundation website. I'll be working on this within the next few weeks, and effort will ramp up further after [Zcon0](https://z.cash.foundation//zcon/).

**Comment by tloriato** [comment]
Why not a bot that relays information back and forth between here & the other proposed place ( i.e forum)?

I've seen it working a few times between IRC (devs) and Slack (users).

**Comment by bitcartel** [comment]
Just ran into a Github UX problem, where community members are having _separate_ discussions even though on the same ticket...

https://forum.z.cash/t/let-s-talk-about-asic-mining/27353/629?u=bitcartel

**Comment by amiller** [comment]
Closing as not an actionable issue, although the discussion could certainly continue. We also have the rocket chat, mailing list, and forum for discussion... https://forum.z.cash/c/foundation I hope the good things about using github this way get to continue.
